cadcdata/cadcdata/tests/data/cadcget_help.txt

```diff
@@ -0,0 +1,39 @@
+usage: cadcget [-h] [--cert CERT | -n | --netrc-file NETRC_FILE | -u USER]
```

Comment:
I don't know if it would be useful to mention that files are automatically decompressed or not?

Reply:
Added

Reply:
No automatic decompression yet.